README.md

```diff
@@ -22,11 +22,13 @@ git clone --branch LATEST_ROS_VERSION https://github.com/ros2/rmw_iceoryx.git
 For alternative installation instructions and more details about iceoryx's internals, please see [iceoryx's GitHub repo](https://github.com/eclipse/iceoryx).
 
 rmw_iceoryx is compatible with ROS 2 starting with Eloquent release.
-Assuming you have ROS2 installed correctly, you can compile the iceoryx workspace with colcon:
+Assuming you have ROS 2 installed correctly, you can compile the iceoryx workspace with colcon:
 
 ```bash
 cd ~/iceoryx_ws/
 source /opt/ros/LATEST_ROS_VERSION/setup.bash  # alternatively source your own ROS 2 workspace
+rosdep update
+rosdep install --from-paths src --ignore-src --rosdistro LATEST_ROS_VERSION -y
```

Comment:
@LanderU Which dependecies will be installed here that were missing for you?

Reply:
@mossmaurice, `ros-ROS_DISTRO-test-msgs`